signer/keydbstore/rethink_keydbstore.go

```diff
@@ -77,7 +79,7 @@ func rdbPrivateKeyFromJSON(data []byte) (interface{}, error) {
 // PrivateKeysRethinkTable is the table definition for notary signer's key information
 var PrivateKeysRethinkTable = rethinkdb.Table{
 	Name:             RDBPrivateKey{}.TableName(),
-	PrimaryKey:       RDBPrivateKey{}.KeyID,
+	PrimaryKey:       "key_id",
```

Comment:
Ah yes, sorry I missed that.  I'm fine with a hardcoded string for now and following up with reflection in a separate PR